apps/mull-api/src/app/event/event.resolver.spec.ts

```diff
@@ -3,13 +3,19 @@ import { EventResolver } from './event.resolver';
 import { EventService } from './event.service';
 import { CreateEventInput, UpdateEventInput } from './inputs/event.input';
 import { mockPartialEvent, mockAllEvents } from './event.mockdata';
+import { User } from '../entities';
 
 const mockEventService = () => ({
   create: jest.fn((mockEventData: CreateEventInput) => ({ ...mockEventData })),
   findOne: jest.fn((id: number) => mockAllEvents.find((event) => event.id === id)),
   findAll: jest.fn(() => mockAllEvents),
   update: jest.fn((mockEventData: UpdateEventInput) => ({ ...mockEventData })),
   delete: jest.fn((id: number) => mockAllEvents.find((event) => event.id === id)),
+  addParticipant: jest.fn((eventId: number, userId: number) => {
```

Comment:
This file contains unit tests, if we called the real eventService it would save the data to the db. Here I'm just testing that my resolver takes in the expected parameters and returns the expected result.

We're not testing database interactions in our codebase currently.

Reply:
Ah thank you! Sorry for having so many questions xD I'm not very familiar with this code base. Not really related to this, just a question. With the codebases I worked with, we have a local test database where we actually write to and clear every time. I'm just wondering if that can be considered in the future?

Reply:
Definitely something we can consider in the future. Maybe when we set up the database for e2e tests we can also set one up for this